src/components/Button.tsx

```diff
@@ -55,6 +52,17 @@ export function Button({
       {endAdornment && <span css={Css.ml1.$}>{endAdornment}</span>}
     </button>
   );
+
+  // If we're disabled b/c of a non-boolean ReactNode, show it in a tooltip
+  if (isDisabled && typeof disabled !== "boolean") {
+    return (
+      <Tooltip title={disabled} delay={100}>
```

Comment:
Nice ☝️! Are you facing any _type_ issues with the **Tooltip** `title` prop (I am not sure if that was updated to accept a ReactNode or just a string.